extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -465,6 +465,9 @@ public function execute() {
 				global $wgNotAValidWikia;
 				$redirect = $wgNotAValidWikia . '?from=' . rawurlencode( $this->mServerName );
 				$this->debug( "redirected to {$redirect}, {$this->mWikiID} {$this->mIsWikiaActive}" );
+				if (  wfHttpsAllowedForURL( $redirect ) && wfHttpsEnabledForURL( $redirect ) && !empty( $_SERVER['HTTP_FASTLY_FF'] ) ) {
```

Comment:
I think `wfHttpsEnabledForURL` is being removed in https://github.com/Wikia/app/pull/17171 so we can probably skip that here since if the URL can support HTTPS, it should be enabled now that domain migration is done.